tests/integration/targets/x509_crl/tasks/impl.yml

```diff
@@ -124,6 +128,101 @@
       - serial_number: 1234
         revocation_date: 20191001000000Z
   register: crl_1_idem_content
+- name: Create CRL 1 (format, check mode)
+  x509_crl:
+    path: '{{ output_dir }}/ca-crl1.crl'
+    privatekey_path: '{{ output_dir }}/ca.key'
+    format: der
+    issuer:
+      CN: Ansible
+    last_update: 20191013000000Z
+    next_update: 20191113000000Z
+    revoked_certificates:
+      - path: '{{ output_dir }}/cert-1.pem'
+        revocation_date: 20191013000000Z
+      - path: '{{ output_dir }}/cert-2.pem'
+        revocation_date: 20191013000000Z
+        reason: key_compromise
+        reason_critical: yes
+        invalidity_date: 20191012000000Z
+      - serial_number: 1234
+        revocation_date: 20191001000000Z
+  check_mode: yes
+  register: crl_1_format_check
```

Comment:
```suggestion
  register: crl_1_format_check
  
```
if there was a blank line before every `- name:`, it would be much (much much) easier to read, in my opinion.

Reply:
Well, right now it fits into how the tests currently are. I'll eventually refactor them so they will be better to read, but for now I'll stick to the current style.

Reply:
yep, it's a good idea to refactor them